types/src/front/lib/assistant.ts

```diff
@@ -78,6 +79,15 @@ export const MISTRAL_7B_DEFAULT_MODEL_CONFIG = {
   largeModel: false,
 } as const;
 
+export const MISTRAL_SMALL_MODEL_CONFIG = {
```

Comment:
This is unrelated to the addition of mistral to core + dev tools let's keep it for the next PR :+1: